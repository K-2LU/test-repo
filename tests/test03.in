200
8138 9711 8955 6727 4031 8024 3570 4148 6078 4780 4200 8208 1446 7602 992 4567 7054 144 8530 8313 1139 4767 2667 9984 4307 55 8670 1391 6439 6442 2783 929 2505 8090 7656 2888 6114 7579 7037 8545 2359 7589 6753 157 1543 7745 4725 8597 7890 9607 3263 5381 727 2282 5366 5034 2337 4036 2778 8777 6831 5561 6058 9336 4 3715 8577 6118 7646 1966 4663 6357 9555 7768 6514 1099 5514 7591 6048 9756 3551 5663 5137 4278 7946 503 5664 283 892 8442 5412 7723 356 7823 3411 360 7890 1988 6478 5536 3954 7494 1893 9862 5262 4759 961 7128 8703 3361 3236 2254 9025 8374 6532 3323 5229 8548 9958 6121 6991 5371 3844 3699 3194 3608 4059 1084 5596 6889 2972 9551 4383 1217 9413 5998 5976 6726 3126 4679 6439 2715 6933 1816 7441 9817 5139 2670 4718 5098 5144 8061 6821 5340 1760 6367 8948 5819 3803 897 9060 6775 448 9796 7992 6213 5794 320 9291 5272 5000 2082 7987 8285 3899 5428 4455 5390 4451 9173 6840 5947 7234 3661 7639 8994 6380 6588 1165 183 7485 225 3310 4285 6373 7654 6850 2167 7975 6141 