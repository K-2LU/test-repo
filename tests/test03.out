168 194 183 130 64 165 55 67 112 82 68 169 25 150 18 75 141 3 174 171 21 81 39 200 71 2 179 24 123 124 43 16 38 167 154 44 113 147 140 175 37 148 131 4 26 156 79 178 159 193 49 95 13 35 93 84 36 65 42 181 134 101 111 189 1 59 177 114 152 30 76 118 192 157 126 20 99 149 110 195 54 103 86 69 161 12 104 7 14 173 97 155 9 158 53 10 159 31 125 100 63 146 29 198 90 80 17 142 180 52 48 34 185 172 127 51 89 176 199 115 139 94 61 58 47 56 66 19 102 137 45 191 72 23 190 109 108 129 46 77 123 41 138 28 144 197 87 40 78 85 88 166 133 92 27 119 182 106 60 15 186 132 11 196 164 117 105 8 188 91 83 32 163 170 62 98 74 96 73 187 135 107 143 57 151 184 121 128 22 5 145 6 50 70 120 153 136 33 162 116